config/routes.rb

```diff
@@ -22,4 +22,6 @@
   patch "/pets/:pet_id/pending", to: "pets#adoption_status_pending"
   patch "/pets/:pet_id/adoptable", to: "pets#adoption_status_adoptable"
 
+  get "shelters/:shelter_id/reviews/new", to: "reviews#new"
```

Comment:
Previously discussed (and just commenting for the sake of comments), but would it make more sense to have `"/reviews/:shelter_id/new"` and `"/reviews/:review_id/edit" paths?